ppr-ui/src/composables/fees/FeeSummary.vue

```diff
@@ -223,6 +236,7 @@ export default defineComponent({
     const localState = reactive({
       feeType: props.setFeeType,
       registrationType: props.setRegistrationType,
+      transferType: props.setTransferType,
```

Comment:
I know this is what has been done elsewhere with props in some older components but unless you are going to mutate this prop value locally, you can use it directly.

Reply:
Gotcha, had it as is just to be consistent. Will change